### PR TITLE
Add Arabic locale

### PR DESCRIPTION
**Pull request by salemaljebaly**
### Add Arabic Localization to ThingsBoard Flutter App

Description:
This PR introduces Arabic translations for the ThingsBoard Flutter app to enhance user experience for Arabic-speaking users. The translations cover all user-facing strings, including navigation items, notifications, forms, and dialogs.

Changes:
1. Added intl_ar.arb file with complete Arabic translations.
2. Utilized the existing translation techniques in the app with minimal modifications.
3. Generate messages_ar.dart file.
4. Add ar locale to LibraryLoader in messages_all.dart file.
5. Add ar to supportedLocales in I10n.dart file.

### CODE REVIEW

**Comment by salemaljebaly** [comment]
I add ar support to app, check it when you have time.
@ybeshkarov

File: lib/l10n/intl_ar.arb [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  "backupCodeAuthPlaceholder": "الرمز الاحتياطي",
+  "verificationCodeIncorrect": "الرمز غير صحيح",
+  "verificationCodeManyRequest": "طلبات كثيرة للتحقق من الرمز",
+  "tryAnotherWay": "حاول بطريقة أخرى",
+  "selectWayToVerify": "اختر طريقة للتحقق",
+  "mfaProviderTopt": "تطبيق المصادقة",
+  "mfaProviderSms": "SMS",
+  "mfaProviderEmail": "البريد الإلكتروني",
+  "mfaProviderBackupCode": "الرمز الاحتياطي"
+}
```
Comment (ybeshkarov) [inline lib/l10n/intl_ar.arb:112]: Add a new line please

**Review by ybeshkarov — CHANGES_REQUESTED** [review]

**Comment by ybeshkarov** [comment]
Looks good!

Could you please point the dev branch as a target? 

Thanks.

File: lib/l10n/intl_ar.arb (hunk 1 above)
Comment (ybeshkarov) [reply lib/l10n/intl_ar.arb:112]: Nope, just add a 113 empty line

**Review by ybeshkarov — APPROVED** [review]

**Pull request merged**